packages/typespec-ts/test/modularIntegration/lroStardard.spec.ts

```diff
@@ -73,15 +72,73 @@ describe("LROStandardClient Classical Client", () => {
           abortSignal: abortController.signal
         }
       );
-      await poller.submitted();
-      assert.strictEqual(poller.operationState?.status, "running");
       abortController.abort();
-      poller.poll();
-      assert.strictEqual(poller.operationState?.status, "cancelled");
+      try {
+        await poller.submitted();
+        assert.fail("Should throw an AbortError");
+      } catch (err: any) {
+        assert.strictEqual(err.message, "The operation was aborted.");
+      }
+    });
+
+    it("should abort polling request", async () => {
+      const abortController = new AbortController();
+      const poller = client.createOrReplace(
+        "madge",
+        {
+          role: "contributor"
+        } as any,
+        {
+          abortSignal: abortController.signal
+        }
+      );
+      try {
+        await poller.submitted();
+        abortController.abort();
+        await poller.poll();
+        assert.fail("Should throw an AbortError");
```

Comment:
is the case here trying to testify abort submit? what would happen if the initial request has submitted successfully ? Also, why would it impact the polling? Also it reminds me of a question, is the abort about LRO itself or about some specific async operation?

Reply:
there are 3 cases, one is to verify the initial request; one is to verify the polling; the other is for the both the initial and polling set and both are set.